templates/02_people.conf.j2

```diff
@@ -1,7 +1,7 @@
 <VirtualHost *:80>
-  ServerName {{ httpd_people_hostname }}
-  CustomLog logs/{{ httpd_people_hostname }}_access.log combined
-  ErrorLog logs/{{ httpd_people_hostname }}_error.log
+  ServerName {{ httpd_www_hostname }}
```

Comment:
Same remark as before : httpd_people_hostname is the correct ansible variable so why changing it ?